packages/babel-helper-define-polyfill-provider/src/utils.ts

```diff
@@ -34,12 +31,16 @@ function resolveId(path) {
   }
 }
 
-export function resolveKey(path: NodePath, computed: boolean = false) {
-  const { node, parent, scope } = path;
-  if (path.isStringLiteral()) return node.value;
-  const { name } = node;
+export function resolveKey(
+  path: NodePath<t.Expression>,
```

Comment:
```suggestion
  path: NodePath<t.Expression | t.PrivateName>,
```
(the type errors about this are correct)